grpc/execution/server.go

```diff
@@ -79,169 +70,112 @@ var (
 	commitmentStateUpdateTimer = metrics.GetOrRegisterTimer("astria/execution/commitment", nil)
 )
 
-func NewExecutionServiceServerV1(eth *eth.Ethereum) (*ExecutionServiceServerV1, error) {
+func NewExecutionServiceServerV2(eth *eth.Ethereum, softAsFirm bool, softAsFirmMaxHeight uint64) (*ExecutionServiceServerV2, error) {
```

Comment:
pass in the `softAsFirmConfig` instead of the two args potentially? if you're ok with exporting it